Python/Product/TestAdapter/PythonStackTraceParser.cs

```diff
@@ -31,7 +31,7 @@ namespace Microsoft.PythonTools.TestAdapter {
     sealed class PythonStackTraceParser : IStackTraceParser {
         public Uri ExecutorUri {
             get {
-                return PythonConstants.ExecutorUri;
+                return PythonConstants.PytestExecutorUri;
```

Comment:
I'd change that for the unittest execution uri. The implementation is for that (unittest).

Reply:
(as far as i know we don't fill in the stack trace for pytest)

Reply:
ya i wasn't sure about this change. Unittest makes more sense